client/src/pages/Followers.js

```diff
@@ -0,0 +1,188 @@
+import { useState, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Grid,
+  makeStyles,
+  Paper,
+  Tab,
+  Tabs,
+  Typography,
+} from "@material-ui/core";
+import AccountCircleIcon from "@material-ui/icons/AccountCircle";
+import axios from "axios";
+
+const useStyles = makeStyles(() => ({
+  followButton: {
+    height: 40,
+    borderRadius: 28,
+    width: 120,
+    marginLeft: "auto",
+  },
+  pfpPlaceholder: {
+    fontSize: 50,
+    marginRight: 10,
+  },
+  //TODO: Needs better name
+  followersPageItem: {
+    width: 750,
+  },
+}));
+
+//TODO: Consider having one list of users and adding an `isFollowing` property. Probably less overhead that way.
```

Comment:
the downside of this is you may have to do filtering to know which users to display in which list so that's overhead as well

Reply:
Good point. I think I'll keep it as is for now since it means less work for me lol